pkg/kubelet/rkt/rkt.go

```diff
@@ -1322,14 +1322,14 @@ func (r *Runtime) getImageByName(imageName string) (*kubecontainer.Image, error)
 
 // ListImages lists all the available appc images on the machine by invoking 'rkt image list'.
 func (r *Runtime) ListImages() ([]kubecontainer.Image, error) {
-	// Example output of 'rkt image list --fields=key,name':
+	// Example output of 'rkt image list --fields=id,name':
 	//
-	// KEY									        NAME
+	// ID									        NAME
 	// sha512-374770396f23dd153937cd66694fe705cf375bcec7da00cf87e1d9f72c192da7	nginx:latest
 	// sha512-bead9e0df8b1b4904d0c57ade2230e6d236e8473f62614a8bc6dcf11fc924123	coreos.com/rkt/stage1:0.8.1
 	//
 	// With '--no-legend=true' the fist line (KEY NAME) will be omitted.
-	output, err := r.runCommand("image", "list", "--no-legend=true", "--fields=key,name")
+	output, err := r.runCommand("image", "list", "--no-legend=true", "--fields=id,name")
```

Comment:
one minor nit, image id is truncated by default now, I'm not how it used later (or will be in future) but potentially can cause a problem, you need to specify `--full` flag to get full output (and generally one should never truncate any data to early)

look at this:
```sh
# v 0.8.1
➜  sudo ./build-rkt-0.8.1/bin/rkt image list --fields key,name
KEY                                                                     NAME
sha512-f8df83fda2ceac2235ab6dc42043d2214aab186b5bd8e963520f17adc592744b busybox:latest

# v 0.9.0 without --full
➜  sudo ./build-rkt-0.9.0/bin/rkt image list --fields id,name
ID                      NAME
sha512-f8df83fda2ce     busybox:latest

# v 0.9.0 with --full
➜  sudo ./build-rkt-0.9.0/bin/rkt image list --fields id,name --full
ID                                                                      NAME
sha512-f8df83fda2ceac2235ab6dc42043d2214aab186b5bd8e963520f17adc592744b busybox:latest
```